music21/figuredBass/notation.py

```diff
@@ -433,6 +560,13 @@ def _reprInternal(self):
                     '++': '##',
                     '+++': '###',
                     '++++': '####',
+                    '\u266f': '#',
```

Comment:
...a place where existing code doesn't match the modern style mentioned above.  :-O  ah well, can't be perfect.  We didn't have good linters in 2008